### PR TITLE
[SMALLFIX] Add javadoc for FileSystemMasterIntegrationTest#ConcurrentRenamer

### PR DESCRIPTION
**Pull request by shizihao123**

### CODE REVIEW

**Comment by shizihao123** [comment]
I can't figure out why it failed , can anybody tell me how to fix this , thank you very much. @ @AmplabJenkins @ifcharming

**Comment by apc999** [comment]
Jenkins, test this please

File: tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java [hunk 1]
```diff
@@ -926,13 +926,25 @@ public void exec(int depth, int concurrencyDepth, AlluxioURI path) throws Except
     }
   }
 
+  /**
+   * This class provides multiple concurrent threads to rename all files in one directory.
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java]: provides -> runs

File: tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java [hunk 2]
```diff
@@ -949,6 +961,14 @@ public Void call() throws Exception {
       return null;
     }
 
+    /**
+     * Executes the process of renaming all files in one directory by multiple concurrent threads.
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java]: Renames all files in one directory using multiple concurrent threads.

File: tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java [hunk 3]
```diff
@@ -949,6 +961,14 @@ public Void call() throws Exception {
...
+     *
+     * @param depth the depth of files to be renamed in one directory
+     * @param concurrencyDepth the concurrency depth of files to be renamed in one directory
+     * @param path the directory of files to be renamed in
+     * @throws Exception if an exception occurs
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java]: we typically don't include exception javadoc for tests unless they are interesting

File: tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java [hunk 4]
```diff
@@ -926,13 +926,25 @@ public void exec(int depth, int concurrencyDepth, AlluxioURI path) throws Except
...
     private AlluxioURI mInitPath;
 
+    /**
+     * Constructs the concurrent renamer.
+     *
+     * @param depth the depth of files to be renamed in one directory
+     * @param concurrencyDepth the concurrency depth of files to be renamed in one directory
+     * @param rootPath  the source root path of the files to be renamed
+     * @param rootPath2 the destination root path of the files to be renamed
+     * @param initPath the directory of files to be renamed in
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java]: how is this different from `rootPath`?

File: tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java [hunk 5]
```diff
@@ -926,13 +926,25 @@ public void exec(int depth, int concurrencyDepth, AlluxioURI path) throws Except
...
     private int mConcurrencyDepth;
     private AlluxioURI mRootPath;
     private AlluxioURI mRootPath2;
     private AlluxioURI mInitPath;
 
+    /**
+     * Constructs the concurrent renamer.
+     *
+     * @param depth the depth of files to be renamed in one directory
+     * @param concurrencyDepth the concurrency depth of files to be renamed in one directory
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java]: What does concurrency depth mean?

**Comment by saltylin** [comment]
@shizihao123 Can you address the comments?

**Comment by shizihao123** [comment]
I will fix that soon! @saltylin @aaudiber

File: tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java (hunk 4 above)
Comment (shizihao123) [inline tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java]: The "initPath" is "under root path" , and I will fix that soon.

File: tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java (hunk 2 above)
Comment (shizihao123) [inline tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java]: okay! I will fix that soon.

File: tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java (hunk 3 above)
Comment (shizihao123) [inline tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java]: Get that, many thanks.

File: tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java (hunk 1 above)
Comment (shizihao123) [inline tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java]: Looks better , I will fix it soon , thanks a lot.

File: tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java (hunk 5 above)
Comment (shizihao123) [inline tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java]: Sometimes we want to try renaming a path when we're not all the way down, which is what
the concurrency depth for, but may be it is a little difficult to explain clearly to readers without knowing about the source  code.

**Pull request merged**